docs/extensibility/resources.md

```diff
@@ -62,11 +62,21 @@ The `list` section defines extra columns available in the list. The format is si
 
 - **path** - _[required]_ contains the path to the data used for the column.
 - **widget** - optional widget used to render the field referred to by the `path` property. By default the value is displayed verbatim. For more information about the available widgets, see [Display widgets](display-widgets.md).
+- **formula** - optional formula used to modify data referred to by the `path` property. In formula we use naming convention: `data.name` instead of just `name`. To see more details about using formulas see [JSONata](https://docs.jsonata.org/overview.html).
 
 ### Example
 
 ```json
-[{ "path": "spec.url" }, { "path": "spec.priority", "widget": "Badge" }]
+[
+  { "path": "spec.url" },
+  { "path": "spec.priority", "widget": "Badge" },
+  { "path": "spec.toppings", "formula": "$join(data.name, ', ')" },
```

Comment:
could we use less custom k8s resources data in the examples? I think it might be easier to explain it to k8s users with k8s resources.